dynatrace-operator-google-marketplace/chart/dynatrace-operator/templates/configmap.yaml

```diff
@@ -471,9 +1406,68 @@ data:
       networkZone: {{ .Values.networkZone }}
       {{- end }}
 
-      {{- if .Values.activeGate.image }}
       activeGate:
+        {{- if .Values.activeGate.image}}
         image: {{ .Values.activeGate.image }}
+        {{- end}}
+        autoUpdate: {{ .Values.activeGate.autoUpdate }}
+
+      oneAgent:
+        {{- if .Values.oneAgent.image }}
+        image: {{ .Values.oneAgent.image }}
+        {{- end }}
+        autoUpdate: {{ .Values.oneAgent.autoUpdate }}
+        {{- if .Values.oneAgent.version }}
+        version: {{ .Values.oneAgent.version }}
+      {{- end }}
+
+      {{- if .Values.classicFullStack.enabled }}
+      classicFullStack:
+        enabled: true
+
+        {{- if .Values.classicFullStack.nodeSelector }}
+        nodeSelector: {{ .Values.classicFullStack.nodeSelector | toYaml | nindent 8 }}
+        {{- end }}
+
+        {{- if .Values.classicFullStack.tolerations }}
+        tolerations: {{ .Values.classicFullStack.tolerations | toYaml | nindent 8 }}
+        {{- end }}
+
+        {{- if .Values.classicFullStack.waitReadySeconds }}
+        waitReadySeconds: {{ .Values.classicFullStack.waitReadySeconds }}
+        {{- end }}
+
+        {{- if .Values.classicFullStack.resources }}
+        resources: {{ .Values.classicFullStack.resources | toYaml | nindent 8}}
+        {{- end }}
+
+        {{- if and (.Values.classicFullStack.args) (ge (len .Values.classicFullStack.args) (1))}}
+        args: {{ .Values.classicFullStack.args | toYaml | nindent 8 }}
+        {{- end }}
+
+        {{- if and (.Values.classicFullStack.env) (ge (len .Values.classicFullStack.env) (1))}}
+        env: {{ .Values.classicFullStack.env | toYaml | nindent 8 }}
+        {{- end }}
+
+        {{- if .Values.classicFullStack.priorityClassName }}
+        priorityClassName: {{ .Values.classicFullStack.priorityClassName }}
+        {{- end }}
+
+        {{- if .Values.classicFullStack.dnsPolicy }}
+        dnsPolicy: {{ .Values.classicFullStack.dnsPolicy }}
+        {{- end }}
+
+        {{- if .Values.classicFullStack.serviceAccountName }}
+        serviceAccountName: {{ .Values.classicFullStack.serviceAccountName }}
```

Comment:
There is a difference between how the service account is set here and, for example, Kubernetes monitoring.

Since the service accounts names are set by GKE, I presume the extra logic for kubemon is also required here.

Reply:
adapted